src/components/ImageView/ImageViewComponent.tsx

```diff
@@ -225,7 +225,10 @@ export class ImageViewComponent extends React.Component<WidgetProps> {
 
         return (
             <ReactResizeDetector handleWidth handleHeight onResize={this.onResize} refreshMode={"throttle"} refreshRate={33}>
-                <div className="image-view-div" style={{gridTemplateColumns: `repeat(${appStore.numImageColumns}, auto)`}}>
+                <div
+                    className="image-view-div"
+                    style={{gridTemplateColumns: `repeat(${appStore.numImageColumns}, ${appStore.overlayStore.viewWidth}px)`, gridTemplateRows: `repeat(${appStore.numImageRows}, ${appStore.overlayStore.viewHeight}px)`}}
```

Comment:
I am seeing region and contour matching tests failing (apparently the rendered region/contour is off by 1 or 2 screen pixel. Wonder if this is related to the changes here where we explicitly set the width and height in px.

Reply:
Based on my manual tests, I think region and contours are still rendered at the right places. If this is the case, we will need to revised the e2e tests a bit (we check pixel rgba values at selected positions in the png images). Before we do the test revision, I would like to know the root cause just in case.